keadm/cmd/keadm/app/cmd/ctl/restart/restart.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2024 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package restart
+
+import "github.com/spf13/cobra"
+
+var (
+	edgeRestartLongDescription  = `"keadm ctl restart" command delete resources in edge node`
+	edgeRestartShortDescription = `restart ctl function to restart resources in edge node`
```

Comment:
```suggestion
	edgeRestartShortDescription = `Restart resources in edge node`
```
Should not delete at edge